### PR TITLE
Add playground3 scenario

### PR DESCRIPTION
**Pull request by matthewfischer**
Playground3 plagues you with container restarts.
Details in the ANSWERs page.

### CODE REVIEW

**Comment by matthewfischer** [comment]
I will rebase this on-top of playground2 after it merges.

File: playground3/README.md
Comment (nimbinatus) [inline playground3/README.md]: Minor nit, but not blocking merge

```suggestion
Wait a bit and check again. The pods keep restarting; can you figure out why?
```

**Review by nimbinatus — APPROVED** [review]
+1, looks good! Thanks!

**Comment by nimbinatus** [comment]
Looks great, @matthewfischer! Go ahead with the rebase, and then I'll merge when you're ready.

**Pull request merged**
